cpp/src/plasma/malloc.c

```diff
@@ -0,0 +1,170 @@
+#include <assert.h>
+#include <stdlib.h>
+#include <stddef.h>
+#include <stdio.h>
+#include <string.h>
+#include <sys/mman.h>
+#include <unistd.h>
+
```

Comment:
Are you potentially modifying this file in https://github.com/ray-project/ray/pull/411?